examples/hello_world_different_sizes.rs

```diff
@@ -0,0 +1,123 @@
+use std::{io::stdout, thread::sleep, time::Duration};
+
+use anyhow::Result;
+use crossterm::{
+    terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
+    ExecutableCommand,
+};
+use ratatui::{
+    prelude::*,
+    widgets::{Block, Borders},
+};
+use tui_big_text::BigTextBuilder;
+
+fn main() -> Result<()> {
+    stdout().execute(EnterAlternateScreen)?;
+    enable_raw_mode()?;
+    let backend = CrosstermBackend::new(stdout());
+    let mut terminal = Terminal::new(backend)?;
+    terminal.clear()?;
+    terminal.draw(|frame| render(frame).expect("failed to render"))?;
+    sleep(Duration::from_secs(25));
```

Comment:
This is quite a long amount to wait. Mostly I took the sleep approach in the hello world example so that it would stick around long enough for a screenshot via VHS (and to avoid having to worry about an event loop, keyboard handling etc.).
```suggestion
    sleep(Duration::from_secs(5));
```

Reply:
sure, I just wanted to make a screenshot for the pull request and increased to time to easily start the screenshot tool of gnome (which requires the captured screen to be visible at startup).

Regarding the other points, I will try to work on as much of them as I can in the next days.